maui-toc.html

```diff
@@ -1085,6 +1085,14 @@
             <li><a href="/maui/TreeView/mvvm">MVVM</a></li>
             <li><a href="/maui/TreeView/item-height-customization">Item Height Customization</a></li>
             <li><a href="/maui/TreeView/right-to-left">Right to left</a></li>        </ul>
+    </li>
```

Comment:
Follow this order.

![image](https://github.com/syncfusion-content/maui-docs/assets/67908880/4714c92a-d145-4b79-80f9-eeb1f9d539a4)

Reply:
Order changed